### ISSUE TITLE
Wrong namespace error in `.rubocop.yml`

### ISSUE THREAD

**Issue by wathigo** [microverseinc/linters-config#38]
There is a problem with `.rubocop.yml` configuration for Ruby projects. 
Am getting this error `.rubocop.yml: Metrics/LineLength has the wrong namespace - should be Layout`.
The issue has been fixed by modifying
```
Metrics/LineLength:
  Max: 120
```
to 
```
Layout/LineLength:
  Max: 120
```

**Comment by nidalaa** [comment]
I can see this change was introduced here https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md#changes-2. 

Nice catch!

These issue has been resolved in two PR:
- https://github.com/microverseinc/linters-config/pull/36/files
- https://github.com/microverseinc/linters-config/pull/37/files
